src/components/ErrorMessage/index.tsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
-import useStyles from './styles';
 import classNames from 'classnames';
-
+import Icon from '../Icons'
 interface IErrorMessage {
   text: string;
+  disabled?: boolean,
   className?: string;
 }
 
-export const ErrorMessage = (props: IErrorMessage) => {
-  const { text, className } = props;
-  const classes = useStyles();
+const ErrorMessage = (props: IErrorMessage) => {
+  const { text, disabled, className, ...rest } = props;
 
-  return <div className={classNames(classes.message, className)}>{text}</div>;
+  return <div
+    {...rest}
+    className={classNames(className, 'flex items-center gap-1 text-sm leading-none  mt-2', {
+      ['text-error']: !disabled,
+      ['text-text-disabled']: disabled,
```

Comment:
You don't really need `[]` in `['text-text-disabled']: disabled`, you can write it like this 
`'text-text-disabled': disabled`. You can also simplify you 2 cases for disabled and not disabled like `disabled ?  'text-text-disabled' : 'text-error'`

Reply:
Removed the "[]". As for using a ternary here, I'd like to keep the conditionals in classNames consistent with more complex cases, and everywhere else we've got the object notation for that